### PR TITLE
Fix for #494

### PR DESCRIPTION
**Pull request by joelbyren**
Making sure we never actively set Renderer to null to prevent this problem.

### CODE REVIEW

**Comment by lextm** [comment]
Can you move "Make most GraphicsPath be fully rectangular" a2507c2924de41e432038e2165cf4230dd02abc3 to another PR?

File: WinFormsUI/Docking/ThemeBase.cs [hunk 1]
```diff
@@ -123,16 +123,20 @@ public virtual void CleanUp(DockPanel dockPanel)
                 var strip = item.Key;
                 var cache = item.Value;
                 if (cache.Key == ToolStripRenderMode.Custom)
-                    if(cache.Value != null)
+                {
```
Comment (lextm) [inline WinFormsUI/Docking/ThemeBase.cs]: When you put brackets around if statement, you should also put them around the corresponding else statement (one common StyleCop rule).

File: WinFormsUI/Docking/ThemeBase.cs [hunk 2]
```diff
@@ -123,16 +123,20 @@ public virtual void CleanUp(DockPanel dockPanel)
...
                         strip.Renderer = cache.Value;
+                }
                 else
                     strip.RenderMode = cache.Key;
             }
 
             _stripBefore.Clear();
             if (_managerBefore.Key == ToolStripManagerRenderMode.Custom)
-                if(_managerBefore.Value != null)
+            {
```
Comment (lextm) [inline WinFormsUI/Docking/ThemeBase.cs]: Same here.

**Comment by joelbyren** [comment]
There, I think I fixed it now

File: WinFormsUI/Docking/VisualStudioToolStripExtender.cs [hunk 3]
```diff
@@ -99,7 +99,10 @@ public void SetStyle(ToolStrip strip, VsVersion version, ThemeBase theme)
             }
 
             if (theme == null)
```
Comment (lextm) [inline WinFormsUI/Docking/VisualStudioToolStripExtender.cs]: Hi, one last place we should add brackets to else statement.

Comment (joelbyren) [inline WinFormsUI/Docking/VisualStudioToolStripExtender.cs]: Fixed

**Pull request merged**